src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const cleanedSourceString = sourceString.replace(/\t+/g, ' ').trim();
+  const styleStrings = cleanedSourceString.split(';');
+
+  return styleStrings.reduce((acc, styleString) => {
+    if (styleString.trim()) {
+      const [property, ...valueParts] = styleString.split(':');
+      const value = valueParts.join(':').trim();
```

Comment:
```suggestion
      const [property, value] = styleString.split(':');
      const preparedValue = value.trim();
```
Probably should work the same